packages/react-components/react-components/etc/react-components.unstable.api.md

```diff
@@ -135,14 +135,14 @@ import { personaClassNames } from '@fluentui/react-persona';
 import { PersonaProps } from '@fluentui/react-persona';
 import { PersonaSlots } from '@fluentui/react-persona';
 import { PersonaState } from '@fluentui/react-persona';
-import { Progress } from '@fluentui/react-progress';
-import { progressClassNames } from '@fluentui/react-progress';
+import { ProgressBar } from '@fluentui/react-progress';
```

Comment:
I believe we should maintain the deprecated `Progress` exports here until `ProgressBar` moves to stable or did the plan change (or maybe I misunderstood)?

Reply:
No I accidentally removed them, I will add them back, good callout